src/data/faq/questions.js

```diff
@@ -25,7 +25,7 @@ const questionsAndAnswers = [
     {
         id: "found-a-bug",
         question: "I found a bug, what do I do?",
-        answer: "Go to our github page and create an issue in the correct repository."
+        answer: "Go to our <a href='${global.githubUrl}'>github page</a> and create an issue in the correct repository."
```

Comment:
There is no safe answer

Reply:
Also please redirect to Ketting here. 99% of the issues belong there.
For that 1%, we can always move them to the correct repo.